CMakeLists.txt

```diff
@@ -99,8 +99,9 @@ find_package(prometheus-cpp REQUIRED)
 if(NOT WITH_RAFT)
     find_package(fmt REQUIRED)
 endif()
-find_package(opentelemetry-cpp REQUIRED)
-
+if(NOT WITH_LIGHT)
+    find_package(opentelemetry-cpp REQUIRED)
```

Comment:
Do we need to wrap all prometheus usage in Knowhere code?

Reply:
Milvus Lite maintains a patch to remove the related code.